src/docs/sdk/features.mdx

```diff
@@ -141,6 +141,20 @@ envelope item, the SDK must discard items larger than the `maxAttachmentSize`. E
 useful because attachments could quickly eat up the users' disk space. Furthermore, Relay has a
 [maximum size for attachments](https://docs.sentry.io/product/relay/options/), and we want to reduce unnecessary requests.
 
+## Screenshots
+
+When the user opts-in, if technically possible, take a screenshot of the application and include as an [attachments](#attachments) to the [envelope](/sdk/envelopes/) with the event.
+
+This feature only applies to SDKs with a user interface, such as Mobile and Desktop. 
+In some environments such as native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. So inherently this feature will be a best effort.
+It's advised to provide this feature through a single option called `attachScreenshot`. That's a preferred way but in platforms such as Flutter, a wrapping widget is required so documentation can point users to that instead of the suggested option name.
+
+The feature is achieved by adding an attachment with:
+
+* File name `screenshot.jpg` or `screenshot.png`
```

Comment:
should we mention that it should overwrite any existing attachment with that given name during the process of attaching the screenshot?

Reply:
its a good point, or doing a sort of send it once and drop it, so the same attachment won't be attached to every following event

Reply:
I think we should just add one and if there's always one added, the front end will just pick one of them. It's an edge case I rather the SDK doesn't have custom code to look for stuff in the attachments list (it can also be added after our check though so it would anwyay be best effort)

Reply:
so if there were 3 events before, the 4th event contains the 3 previous attachments + the new one? I'd say it's burning users' data and not that nice.

Reply:
The SDK should add a single attachment when capturing the event. It'll not keep adding to the scope. Each capture includes a fresh screenshot. If the user had already added one manually with the same name, that's just an edge case I don't think worth spending too many cycles on.